2022-07-31 14:07:35,981 INFO    MainThread:9136 [internal.py:wandb_internal():90] W&B internal server running at pid: 9136, started at: 2022-07-31 14:07:35.980791
2022-07-31 14:07:35,983 INFO    WriterThread:9136 [datastore.py:open_for_write():75] open: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\run-22lcfsam.wandb
2022-07-31 14:07:35,989 DEBUG   SenderThread:9136 [sender.py:send():236] send: header
2022-07-31 14:07:35,989 DEBUG   SenderThread:9136 [sender.py:send():236] send: run
2022-07-31 14:07:36,722 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: check_version
2022-07-31 14:07:36,723 INFO    SenderThread:9136 [dir_watcher.py:__init__():216] watching files in: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files
2022-07-31 14:07:36,723 INFO    SenderThread:9136 [sender.py:_start_run_threads():815] run started: 22lcfsam with start time 1659247652
2022-07-31 14:07:36,724 DEBUG   SenderThread:9136 [sender.py:send():236] send: summary
2022-07-31 14:07:36,724 INFO    SenderThread:9136 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2022-07-31 14:07:36,725 DEBUG   SenderThread:9136 [sender.py:send_request():250] send_request: check_version
2022-07-31 14:07:37,728 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\wandb-summary.json
2022-07-31 14:07:41,319 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: run_start
2022-07-31 14:07:41,387 DEBUG   HandlerThread:9136 [meta.py:__init__():35] meta init
2022-07-31 14:07:41,387 DEBUG   HandlerThread:9136 [meta.py:__init__():49] meta init done
2022-07-31 14:07:41,387 DEBUG   HandlerThread:9136 [meta.py:probe():209] probe
2022-07-31 14:07:41,432 DEBUG   HandlerThread:9136 [meta.py:_setup_git():199] setup git
2022-07-31 14:07:41,527 DEBUG   HandlerThread:9136 [meta.py:_setup_git():206] setup git done
2022-07-31 14:07:41,527 DEBUG   HandlerThread:9136 [meta.py:_save_pip():53] save pip
2022-07-31 14:07:41,528 DEBUG   HandlerThread:9136 [meta.py:_save_pip():67] save pip done
2022-07-31 14:07:41,528 DEBUG   HandlerThread:9136 [meta.py:_save_conda():74] save conda
2022-07-31 14:07:41,728 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\conda-environment.yaml
2022-07-31 14:07:41,729 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\requirements.txt
2022-07-31 14:07:44,974 DEBUG   HandlerThread:9136 [meta.py:_save_conda():84] save conda done
2022-07-31 14:07:44,974 DEBUG   HandlerThread:9136 [meta.py:probe():247] probe done
2022-07-31 14:07:45,234 DEBUG   SenderThread:9136 [sender.py:send():236] send: files
2022-07-31 14:07:45,234 INFO    SenderThread:9136 [sender.py:_save_file():952] saving file wandb-metadata.json with policy now
2022-07-31 14:07:45,235 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:07:45,237 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: pause
2022-07-31 14:07:45,237 INFO    HandlerThread:9136 [handler.py:handle_request_pause():695] stopping system metrics thread
2022-07-31 14:07:45,237 DEBUG   SenderThread:9136 [sender.py:send_request():250] send_request: stop_status
2022-07-31 14:07:45,641 DEBUG   SenderThread:9136 [sender.py:send():236] send: telemetry
2022-07-31 14:07:45,641 DEBUG   SenderThread:9136 [sender.py:send():236] send: stats
2022-07-31 14:07:45,729 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_modified():286] file/dir modified: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\conda-environment.yaml
2022-07-31 14:07:45,730 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\wandb-metadata.json
2022-07-31 14:07:45,730 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\output.log
2022-07-31 14:07:46,246 INFO    Thread-17 :9136 [upload_job.py:push():138] Uploaded file C:\Users\znk\AppData\Local\Temp\tmp0e29mq_twandb\11587vuf-wandb-metadata.json
2022-07-31 14:08:00,643 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:08:00,643 DEBUG   SenderThread:9136 [sender.py:send_request():250] send_request: stop_status
2022-07-31 14:08:06,749 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_modified():286] file/dir modified: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\config.yaml
2022-07-31 14:08:16,024 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:08:16,025 DEBUG   SenderThread:9136 [sender.py:send_request():250] send_request: stop_status
2022-07-31 14:08:31,433 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:08:31,433 DEBUG   SenderThread:9136 [sender.py:send_request():250] send_request: stop_status
2022-07-31 14:08:34,540 ERROR   SenderThread:9136 [retry.py:__call__():132] Retry attempt failed:
Traceback (most recent call last):
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\connectionpool.py", line 700, in urlopen
    self._prepare_proxy(conn)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\connectionpool.py", line 996, in _prepare_proxy
    conn.connect()
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\connection.py", line 364, in connect
    self.sock = conn = self._connect_tls_proxy(hostname, conn)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\connection.py", line 499, in _connect_tls_proxy
    socket = ssl_wrap_socket(
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\util\ssl_.py", line 453, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(sock, context, tls_in_tls)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\util\ssl_.py", line 495, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\ssl.py", line 1040, in _create
    self.do_handshake()
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:1131)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\adapters.py", line 489, in send
    resp = conn.urlopen(
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\connectionpool.py", line 787, in urlopen
    retries = retries.increment(
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\wandb\sdk\lib\retry.py", line 108, in __call__
    result = self._call_fn(*args, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\wandb\sdk\internal\internal_api.py", line 158, in execute
    return self.client.execute(*args, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\transport\requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\api.py", line 115, in post
    return request("post", url, data=data, json=json, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "c:\Users\znk\anaconda3\envs\kaggle\lib\site-packages\requests\adapters.py", line 563, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2022-07-31 14:08:51,434 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:08:52,568 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:08:54,639 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:08:59,647 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:09:08,763 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:09:11,435 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:09:17,276 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: resume
2022-07-31 14:09:17,276 INFO    HandlerThread:9136 [handler.py:handle_request_resume():686] starting system metrics thread
2022-07-31 14:09:23,816 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_modified():286] file/dir modified: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\output.log
2022-07-31 14:09:27,788 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:09:31,436 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:09:51,207 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: log_artifact
2022-07-31 14:09:51,437 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:09:51,848 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\media\table\df_train_0_14d58ec5629b1f76ce55.table.json
2022-07-31 14:09:51,848 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\media
2022-07-31 14:09:51,849 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_created():269] file/dir created: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\media\table
2022-07-31 14:09:55,852 INFO    Thread-11 :9136 [dir_watcher.py:_on_file_modified():286] file/dir modified: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\files\output.log
2022-07-31 14:10:04,346 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:10:11,438 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:10:31,439 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:10:51,440 DEBUG   HandlerThread:9136 [handler.py:handle_request():140] handle_request: stop_status
2022-07-31 14:10:57,106 WARNING MainThread:9136 [internal.py:is_dead():387] Internal process exiting, parent pid 9528 disappeared
2022-07-31 14:10:57,106 ERROR   MainThread:9136 [internal.py:wandb_internal():149] Internal process shutdown.
2022-07-31 14:10:57,162 INFO    WriterThread:9136 [datastore.py:close():279] close: c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\run-20220731_140731-22lcfsam\run-22lcfsam.wandb
2022-07-31 14:10:57,457 INFO    HandlerThread:9136 [handler.py:finish():806] shutting down handler
2022-07-31 14:11:20,650 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:13:46,384 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:19:05,264 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:25:12,766 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:31:19,068 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:36:49,458 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:42:47,317 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:48:22,831 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:53:37,467 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 14:58:57,404 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:04:32,012 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:09:49,499 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:15:08,147 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:20:39,375 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:26:26,580 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:32:02,366 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:37:56,628 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:43:21,704 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:49:25,033 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 15:55:30,886 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:00:49,439 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:06:47,862 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:11:56,879 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:17:36,333 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:23:49,309 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:23:51,484 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:23:55,605 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:24:05,471 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:24:23,418 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:25:00,703 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:26:16,971 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:28:47,909 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:33:59,842 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:39:50,802 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:45:44,084 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:51:32,907 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 16:57:44,519 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:03:07,250 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:08:13,081 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:13:29,970 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:19:00,404 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:24:55,854 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:30:58,460 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:36:56,130 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:42:42,304 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:48:28,761 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 17:54:41,832 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:00:47,107 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:06:18,024 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:12:03,179 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:17:21,932 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:23:00,225 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:28:39,948 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:34:08,616 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:40:13,709 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:40:15,897 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:40:23,274 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:40:31,906 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:40:50,759 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:41:29,328 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:42:40,675 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:45:19,950 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:49:44,220 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 18:55:59,168 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:01:57,804 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:07:52,494 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:13:10,415 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:18:44,990 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:24:38,906 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:30:18,596 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:35:26,558 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:41:00,486 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:47:07,891 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:52:57,763 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 19:58:16,880 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:03:52,808 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:09:18,362 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:14:44,064 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:20:41,064 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:26:14,768 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:31:59,493 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:37:28,783 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:43:25,015 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:49:21,568 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:55:55,251 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:55:57,550 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:56:01,843 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:56:11,736 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:56:30,449 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:57:07,890 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 20:58:17,648 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:00:26,510 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:05:24,743 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:10:33,941 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:16:40,114 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:21:51,955 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:26:55,705 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:32:13,187 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:37:26,212 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:42:41,812 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:48:32,684 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 21:54:30,539 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:00:13,879 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:06:23,379 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:11:49,580 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:17:38,462 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:22:44,166 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:28:23,523 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:34:12,461 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:39:15,486 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:44:52,710 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:51:05,685 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 22:56:43,972 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:02:09,393 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:08:35,324 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:08:37,762 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:08:42,290 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:08:51,125 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:09:10,761 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:09:48,153 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:11:01,234 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:13:19,864 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:17:57,958 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:23:45,627 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:29:39,922 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:35:05,209 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:40:55,968 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:46:30,023 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:51:51,938 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2022-07-31 23:57:34,156 WARNING FileStreamThread:9136 [file_stream.py:request_with_retry():589] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/huc/hubmap-organ-segmentation/22lcfsam/file_stream (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001FD85A68700>>, args: ('https://api.wandb.ai/files/huc/hubmap-organ-segmentation/22lcfsam/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
